clients/nexus-iq/src/main/kotlin/NexusIqService.kt

```diff
@@ -34,6 +34,7 @@ import retrofit2.converter.jackson.JacksonConverterFactory
 import retrofit2.http.Body
```

Comment:
Commit message: Maybe add a remark for what purpose this information is going to be used. I was wondering why these additional endpoints are needed. (Also for the previous commit.)

Reply:
These are for my own experiments (that are not related to ORT), but I chose to simply add them here and use Kotlin script instead of writing a separate Python script or so.

Reply:
If it's for an experiment, does it have to be merged? Couldn't it just stay on a branch until the experiment is done?
Apart from that I agree with @oheger-bosch that the "why" should be answered in the commit message, maybe just something like "for completeness, because they might be used later".

Reply:
> "for completeness, because they might be used later"

But isn't that obvious, as we always said the libraries in `clients` should be *generic* clients, not necessarily limited to ORT's needs (and especially not depend on ORT)?

Reply:
I didn't know we made that clear statement, but ok.

Reply:
I've anyway amended the commit messages.

Reply:
> I didn't know we made that clear statement, but ok.

See [clients/README.md](https://github.com/oss-review-toolkit/ort/blob/master/clients/README.md).